VERSION

```diff
@@ -1 +1 @@
-0.5.3
+0.5.4
```

Comment:
Given this PR exposes changes to the end user, I think you should up a minor to `0.6.0` instead of a patch

Reply:
+1 to that!